ds_judgements_public_ui/templates/includes/results_filters_inputs.html

```diff
@@ -1,93 +1,61 @@
-{% load court_utils errors %}
+{% load court_utils errors search_results_filters crispy_forms_tags %}
 <div class="structured-search__multi-fields-panel">
   <div class="structured-search__limit-to-container">
-    <fieldset class="{% if context.errors|has_errors_for_field:"from_date" %}with-errors{% endif %}">
+    <fieldset class="{% if form.non_field_errors %}with-errors{% endif %}">
       <legend class="structured-search__limit-to-label">From date</legend>
       <p class="structured-search__help-text" id="from_date-help-text">
         For example <em>01 01 2003</em> or <em>2003</em>
       </p>
-      {% if context.errors|has_errors_for_field:"from_date" %}
+      {% if form.non_field_errors %}
```

Comment:
This handling of the data fourt times feels unpleasant, but I don't know a better way to handle it.

Reply:
Have reduced this greatly, so shouldn't be quite as horrible now!